### DISCUSSION TITLE
SimpleTcp vs SuperSocket

### DISCUSSION THREAD

**Discussion by mzelecom** [jchristn/SuperSimpleTcp#40] [General]
Hi.
i used SuperSocket for 1 year and it was good but no update for .netframework.
https://www.nuget.org/packages/SuperSocket/
should i change SuperSocket  to SimpleTcp?
is SimpleTcp better than SuperSocket?

**Comment by jchristn** [comment]
Hi @mzelecom thanks for your note.  In general I wouldn't recommend switching libraries unless the library you are using is no longer meeting your needs, has issues that aren't being resolved, or doesn't support things which you can forsee yourself needing.  I don't personally have experience using SuperSocket, so I can't speak directly to a comparison, but it does appear to offer a broad set of services and good community supporting it.  Also, I wouldn't measure the investment into .NET Framework; most open source developers have begun phasing it out of their packages in favor of .NET Standard, Core, and 5.0.  Cheers, Joel
